modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/saml/AbstractSAMLSSOTestCase.java

```diff
@@ -403,6 +420,16 @@ public SAMLSSOServiceProviderDTO createSsoSPDTOForIdPInit(SAMLConfig config){
         return idpInitSpDTO;
     }
 
+    /**
```

Comment:
Missing description

Reply:
Fixed https://github.com/wso2/product-is/pull/9477/commits/70d73b570425b27e4ff199a8c93aec0c7fb47645